packages/driver/cypress/e2e/commands/local_storage.cy.ts

```diff
@@ -0,0 +1,325 @@
+        assertLogLength(logs, 1)
+        expect(logs[0].get('name')).to.eq('visit')
+      })
+    })
+  })
+
+  context('#getAllSessionStorage', () => {
+    beforeEach(() => {
+      cy.visit('/fixtures/set-storage-on-multiple-origins.html')
+    })
+
+    it('gets local storage from all origins', () => {
+      cy.getAllSessionStorage().should('deep.equal', {
+        'http://localhost:3500': {
+          key11: 'value11',
+          key12: 'value12',
+        },
+        'http://www.foobar.com:3500': {
+          key13: 'value13',
+          key14: 'value14',
+        },
+        'http://other.foobar.com:3500': {
+          key15: 'value15',
+          key16: 'value16',
+        },
+        'http://barbaz.com:3500': {
+          key17: 'value17',
+          key18: 'value18',
+        },
+      })
+    })
+
+    it('logs once', () => {
+      cy.getAllSessionStorage().then(() => {
+        assertLogLength(logs, 2)
+        expect(logs[0].get('name')).to.eq('visit')
+        expect(logs[1].get('name')).to.eq('getAllSessionStorage')
+      })
+    })
+
+    it('does not log when log: false', () => {
+      cy.getAllSessionStorage({ log: false }).then(() => {
+        assertLogLength(logs, 1)
+        expect(logs[0].get('name')).to.eq('visit')
+      })
+    })
+
+    it('consoleProps includes the storage yielded', () => {
+      cy.getAllSessionStorage().then(() => {
+        const consoleProps = logs[1].get('consoleProps')()
+
+        expect(consoleProps).to.deep.equal({
+          Command: 'getAllSessionStorage',
+          Yielded: {
+            'http://localhost:3500': {
+              key11: 'value11',
+              key12: 'value12',
+            },
+            'http://www.foobar.com:3500': {
+              key13: 'value13',
+              key14: 'value14',
+            },
+            'http://other.foobar.com:3500': {
+              key15: 'value15',
+              key16: 'value16',
+            },
+            'http://barbaz.com:3500': {
+              key17: 'value17',
+              key18: 'value18',
+            },
+          },
+        })
+      })
+    })
+  })
+
+  context('#clearAllSessionStorage', () => {
+    beforeEach(() => {
+      cy.visit('/fixtures/set-storage-on-multiple-origins.html')
+    })
+
+    it('clears session storage for all origins', () => {
+      cy.clearAllSessionStorage()
+      cy.getAllSessionStorage().should('deep.equal', {})
+    })
+
+    it('logs once', () => {
+      cy.clearAllSessionStorage().then(() => {
+        assertLogLength(logs, 2)
+        expect(logs[0].get('name')).to.eq('visit')
+        expect(logs[1].get('name')).to.eq('clearAllSessionStorage')
+      })
+    })
+
+    it('does not log when log: false', () => {
+      cy.clearAllSessionStorage({ log: false }).then(() => {
+        assertLogLength(logs, 1)
+        expect(logs[0].get('name')).to.eq('visit')
+      })
+    })
+  })
+
+  context('#clearLocalStorage', () => {
```

Comment:
I converted this file from `.js` to `.ts`. The below tests were the only existing ones in the `.js` file and I left them unchanged outside of a couple type annotations.